### PR TITLE
Ensure cookie crumbs wrap on small screens

### PR DESCRIPTION
**Pull request by darkwing**
Check any individual loan detail, size your browser down.

### CODE REVIEW

**Comment by pdehaan** [comment]
One small nit, at medium resolutions, the breadcrumbs are getting a bit too "familiar" with the right edge of the screen. 

runs_ _loads_app

**Comment by pdehaan** [comment]
But otherwise, looks great at smaller viewports!

runs_ _loads_app

**Comment by pdehaan** [comment]
Another small point (unrelated to this PR, so feel free to address separately).
I couldn't figure out how to regenerate the CSS using Stylus, so I had to troll through an older PR.

We may want to add an npm script as a friendly helper:

```sh
$ ./node_modules/.bin/stylus --compress client/static/assets/stylus/app.styl --out client/static/assets/css/
```

So in the package.json scripts section we can add something like this (where the "./node_modules/.bin" isn't needed since npm automatically resolves that when running as a script):
```
"stylus": "stylus --compress client/static/assets/stylus/app.styl --out client/static/assets/css/"
```

Then, all I need to do to regenerate the Stylus CSS is run `npm run stylus` and behold the magic.

File: client/static/assets/stylus/app.styl [hunk 1]
```diff
@@ -33,7 +33,15 @@ nav.breadcrumbs {
   margin: 10px 0;
   padding: 0;
   border: 0;
-  float: right;
+  
+  position: absolute;
+  right: 0;
```
Comment (pdehaan) [inline client/static/assets/stylus/app.styl]: Should this be `8px` or something, so it doesn't touch?

**Comment by pdehaan** [comment]
Apart from that 1.5 comments, looks good to me. :+1: 
Merge when ready, or reassign to me if you want me to look again (or want me to add more GIFs).

**Comment by pdehaan** [comment]
Re: https://github.com/loads/loads-web/pull/36#issuecomment-70725755 (adding `stylus` npm script which converts the .styl files to vanilla .css),
Fixed in #39

**Comment by darkwing** [comment]
Updated!  Merging!

**Pull request merged**